{{cookiecutter.github_repository}}/circle.yml

```diff
@@ -0,0 +1,104 @@
+## Customize the test machine
+machine:
+  python:
+    version: {{ '3.5' if cookiecutter.python3.lower() == 'y' else '2.7.4'}} # 2.7 will cause an error with py.test
+
+  #services: # By default popular ones are running on our build machines (bound to localhost), including Postgres, MySQL, Redis and MongoDB
+  # -redis 
+  #  
+
+  # Add some environment variables
+ # environment:
+ #   DATABASE_URL: postgres://ubuntu:@127.0.0.1:5432/circle_test
+
+## Customize checkout
+#checkout:
+#  post:
+#    - git submodule update --init # use submodules
+
+## Customize dependencies
+dependencies:
+  pre:
+    - sudo service postgresql stop && sudo apt-get remove -y
+      postgresql-9.4 && sudo apt-get update; sudo apt-get install -y
+      postgresql-9.3 postgresql-contrib-9.3
+    - sudo sed -i "s/\port = 5433/port = 5432/"
+      /etc/postgresql/9.3/main/postgresql.conf
+    - sudo cp /etc/postgresql/9.4/main/pg_hba.conf
+      /etc/postgresql/9.3/main/pg_hba.conf
+    - sudo service postgresql restart
+    - sudo -u postgres createuser ubuntu -d --superuser
+    - createdb circle_test
+    - createdb ubuntu
+
+  override:
+    - pip install --upgrade pip
+    - pip install pip-accel
+    - pip-accel install ansible
+    - pip-accel install -r requirements/development.txt
```

Comment:
- You can remove the `pre` block there is no point removing 9.4 and adding 9.3
- use the default postgres installation, just `export DATABASE_URL=postgres://ubnutu@localhost/circle_test`
- circle ci caches the virtualenv by default, `pip-accel` is not really needed. 
- You can remove the `post` block and `cache_directories` as well.
- You can remove the top level `circle.yml` which test the `django-init`, only have the circle.yml config for generated project.